frontend/src/upload.js

```diff
@@ -236,13 +275,14 @@ $(document).ready(function() {
         localStorage.removeItem(file.fileId);
         $(expiry).parents('tr').remove();
         window.clearTimeout(t);
+        toggleHeader();
       }
     }
 
     // create popup
     popupDiv.classList.add('popup');
     $popupText.html(
-      '<span class="del-file">Delete</span><span class="nvm" > Nevermind</span>'
+      '<span class=\'del-file\'>Delete</span><span class=\'nvm\' > Nevermind</span>'
```

Comment:
I think we can also tweak the ESLint `quotes` rule to allow escaping of strings:

```yaml
quotes: [error, single, {avoidEscape: true}]
```

Or in this case since it's HTML strings, just use double quotes on the internal attributes.

```html
'<span class="del-file">Delete</span>&nbsp;<span class="nvm">Nevermind</span>'
```